### PR TITLE
[release-4.10] fix [sriov] Bond CNI integration bond cni over sriov test

### PR DESCRIPTION
**Pull request by zeeke**
Manual backport of
 - https://github.com/openshift-kni/cnf-features-deploy/pull/1207

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/vrf/vrf_sriov.go [hunk 1]
```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/networks"
```
Comment (SchSeba) [inline cnf-tests/testsuites/e2esuite/vrf/vrf_sriov.go:9]: please move the import down to the cnf-tests section

File: cnf-tests/testsuites/pkg/namespaces/namespaces.go [hunk 2]
```diff
@@ -2,14 +2,14 @@ package namespaces
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"strings"
 	"time"
 
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
```
Comment (SchSeba) [inline cnf-tests/testsuites/pkg/namespaces/namespaces.go:9]: please move the import to the k8s section

File: cnf-tests/testsuites/e2esuite/vrf/vrf_sriov.go (hunk 1 above)
Comment (zeeke) [reply cnf-tests/testsuites/e2esuite/vrf/vrf_sriov.go:9]: removed, as it wasn't actually used

**Pull request merged**